### PR TITLE
Markdown Lint, Update Mapping Instructions and Server Properties

### PR DESCRIPTION
**Pull request by zanix**
- Updated documentation for overviewer recommending the use of binaries or building instead of using apt
- Updated the example output of `mscs status`
- Updated global and per-server properties to match script
  - Add help for properties that set replacement variables
- Capitalize page titles and headers
- Clean up markdown using remark-lint with the the `remark-preset-lint-recommended` ruleset
  - Replace indented code blocks with code fences
  - Wrap lines to 120
  - Replace most links with footer references
  - Add syntax highlighting to code blocks
- Update installing a Forge server documentation

### CODE REVIEW

**Comment by zanix** [comment]
I don't know how to update the docs site, so @Roflicide, can you do that?

**Comment by zanix** [comment]
Updated the forge server installation documentation

File: docs/mscs/adjusting-world-server-properties/forge.md [hunk 1]
```diff
@@ -59,18 +70,31 @@ If the server fails to start, the `eula.txt` file may need to be edited and acce
 editor /opt/mscs/worlds/forge/eula.txt
 ```
 
+Change `false` to `true`
+
+```ini
+elua=true
```
Comment (sandain) [inline docs/mscs/adjusting-world-server-properties/forge.md]: eula=true

**Review by sandain — COMMENTED** [review]
Besides the minor typo, this looks good. Thanks

File: docs/mscs/adjusting-world-server-properties/forge.md (hunk 1 above)
Comment (zanix) [reply docs/mscs/adjusting-world-server-properties/forge.md]: Fixed

**Comment by zanix** [comment]
Looks like merging into main updates the docs automatically, nice!

**Pull request merged**